bandersnatch_vrfs/Cargo.toml

```diff
@@ -19,8 +19,6 @@ ark-ff.workspace = true
 ark-ec.workspace = true
 ark-serialize.workspace = true
 
-ark-scale = { worspace = true, features = [ "ec" ] }
```

Comment:
This was very nasty.  Rust builds two versions of ark-scale, one for dleq_vrf and one for bandersnatch_vrf, despite it being a workspace dependency with the same features.  It's maybe a cargo/rustc bug?  Or more likely I'm forcing some version number difference elsewhere somehow?  

Anyways if I do not use reexports then this impl becomes invisible outside dleq_vrf:
https://github.com/w3f/ring-vrf/blob/master/dleq_vrf/src/scale.rs#L104-L116